xbdd-web/src/modules/summary/productList/ProductList.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { List, Typography } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+import ProductListStyles from "./styles/ProductListStyles";
+import ProductListItem from "./ProductListItem";
+
+const ProductList = props => {
+  if (!props.list) {
+    return null;
+  } else {
```

Comment:
This doesn't need to be in an else block, just have the return as it reads nicer